env/opal.config.sample.js

```diff
@@ -23,6 +23,8 @@ const config = {
         serviceStatusURL: "./content/service-status.json",
         // Boolean: whether to show the app's version and build number on the front page.
         showVersionOnInit: true,
+        // String: comma-separated list of languages supported in the system (ISO 639-1 codes) with the first language being the default
+        supportedLanguages: 'en, fr',
```

Comment:
Should this be without spaces after the comma?

Reply:
I do sanitize the value to remove spaces, but I'll take the space out of this sample just to keep it simple.